### PR TITLE
update form to cater for university info

### PR DESCRIPTION
**Pull request by DawoudSheraz**
### Description

- Adding universities & campus information through pre-defined choices 
- Added an optional field to take university information, if not present in the list
   - This field is hidden by default, but present in HTML. A custom JS can show it to the user if the user-desired information is not present in the dropdowns.

### CODE REVIEW

File: freshgrad_test/constants.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    ('Virtual University', _('Virtual University')),
+    ('Lahore University of Management & Sciences', _('Lahore University of Management & Sciences')),
+    ('University of Sargodha', _('University of Sargodha')),
+    ('International Islamic University', _('International Islamic University')),
+    ('Other', _('Other'))
+)
+
+
+CAMPUS = (
+    ('Lahore', _('Lahore')),
```
Comment (ziafazal) [inline freshgrad_test/constants.py:43]: nit: no need to mark these strings for internationalization.

File: freshgrad_test/models.py [hunk 2]
```diff
@@ -29,7 +31,11 @@ class CandidateInfo(models.Model):
     """
     user = models.OneToOneField(USER_MODEL)
     cnic_number = models.CharField(max_length=20, verbose_name=_("CNIC Number"))
-    university = models.CharField(max_length=100)
+    university = models.CharField(max_length=60, choices=UNIVERSITIES)
```
Comment (ziafazal) [inline freshgrad_test/models.py:34]: nit: I think we should keep max_length of 100

**Review by ziafazal — APPROVED** [review]
👍

**Comment by DawoudSheraz** [comment]
I will be addressing the nits, along with backend validation, in the follow-up PR. Merging this PR.

**Pull request merged**